helix-term/src/ui/editor.rs

```diff
@@ -976,15 +976,16 @@ impl EditorView {
                         doc.set_selection(view_id, Selection::point(pos));
                     }
 
-                    editor.focus(view_id);
+                    editor.tree.focus = view_id;
```

Comment:
It looks like the changes of https://github.com/helix-editor/helix/commit/e4c9d4082a139aac3aea4506918171b96e81f5b9 are reverted here. Is that a mistake from a merge conflict or intentional?

Reply:
That's really odd... Definitely not intentional, but I can't imagine how I would've blown away the changes in a conflict.

In any case, I've restored those changes and I don't see anything else unintentional in the remaining changes. Thanks for catching that!